### PR TITLE
Move forwarder extension methods

### PR DESCRIPTION
**Pull request by Kahbazi**
Fixes https://github.com/microsoft/reverse-proxy/issues/1172

### CODE REVIEW

File: src/ReverseProxy/Forwarder/RequestTransformer.cs [hunk 1]
```diff
@@ -1,13 +1,9 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
```
Comment (MihaZupan) [inline src/ReverseProxy/Forwarder/RequestTransformer.cs:3]: The license note should stay

**Review by alnikola — CHANGES_REQUESTED** [review]
Please address @MihaZupan's [comment](https://github.com/microsoft/reverse-proxy/pull/1197#discussion_r694045773). It's important.

**Review by Tratcher — APPROVED** [review]
After fixing the license header.

File: src/ReverseProxy/Forwarder/RequestTransformer.cs [hunk 2]
```diff
@@ -1,13 +1,9 @@
...
 using System;
```
Comment (Kahbazi) [inline src/ReverseProxy/Forwarder/RequestTransformer.cs:1]:
```suggestion
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

using System;
```

**Review by MihaZupan — APPROVED** [review]

**Review by alnikola — APPROVED** [review]

**Comment by alnikola** [comment]
Thanks!

**Pull request merged**